app/assets/javascripts/dataTables/dataTables.bootstrap.js

```diff
@@ -56,8 +56,8 @@ $.extend( true, DataTable.defaults, {
 /* Default class modification */
 $.extend( DataTable.ext.classes, {
 	sWrapper:      "dataTables_wrapper form-inline dt-bootstrap",
-	sFilterInput:  "form-control input-sm",
-	sLengthSelect: "form-control input-sm",
+	sFilterInput:  "datatable-form-control input-sm",
+	sLengthSelect: "datatable-form-control input-sm",
```

Comment:
This is an okay change. It has no effect on my display, but does apply that class to the length and filter select.